tests/test_util.py

```diff
@@ -203,3 +203,31 @@ def test_nonuniform_scale_samples_truncnorm():
         ]
     )
     np.testing.assert_allclose(actual, expected)
+
+
+def test_nonuniform_scale_samples_logunif():
+    """
+    Test the rescaling of samples for logarithmic uniform  distribution
+    """
+    problem = {
+        "num_vars": 1,
+        "dists": ["logunif"],
+        "bounds": [[1e-22, 1e-18]],
+        "names": ["x1"],
+    }
+    actual = latin.sample(problem, 10, seed=2)
+    expected = np.array(
+        [
+            [2.57258950e-22],
+            [1.49416294e-22],
+            [2.08863172e-19],
+            [3.03290952e-20],
+            [1.04680308e-21],
+            [5.09015386e-19],
+            [2.36662233e-21],
+            [1.11611357e-19],
+            [1.35560739e-20],
+            [5.86336757e-21],
+        ]
+    )
+    np.testing.assert_allclose(actual, expected)
```

Comment:
A better test would be to check that the sample does follow the prescribed distribution. This can be done for instance with a fit (and compare the fitted params) or even a classical 2 sample test like Cramer or KS. This would be a more robust test and would not need seeding even.